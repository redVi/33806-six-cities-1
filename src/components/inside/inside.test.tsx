import React from 'react';
import renderer from 'react-test-renderer';
import Inside from '@/components/inside/inside';

describe(`Inside`, () => {
  it(`renders correctly`, () => {
    const tree = renderer.create(
      <Inside title="What&apos;s inside" items={[`Breakfast`, `Towels`]}/>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
