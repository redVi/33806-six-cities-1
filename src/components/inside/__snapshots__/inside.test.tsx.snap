// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Inside renders correctly 1`] = `
<div
  className="property__inside"
>
  <h2
    className="property__inside-title"
  >
    What's inside
  </h2>
  <ul
    className="property__inside-list"
  >
    <li
      className="property__inside-item"
    >
      Breakfast
    </li>
    <li
      className="property__inside-item"
    >
      Towels
    </li>
  </ul>
</div>
`;
